docs/_static/scroll_sidebar.js

```diff
@@ -2,11 +2,23 @@
 
 window.addEventListener("DOMContentLoaded", main);
 
+const url = new URL(window.location.href);
+
 function main() {
+    let num = 0;
+
+    if (url.host.startsWith("disnake--") && url.host.endsWith(".org.readthedocs.build")) {
+        // when building docs for PRs, rtd inserts the fucking
+        // "warning" telling that the page was created from PR
+        // just before the main ul, thus we need to increment
+        // this to skip admonition and get ul instead of div
+        num = 1;
+    }
+
```

Comment:
this seems a bit hacky and could stop working for any reason, we should try to not do this.

Reply:
hmm doesn't make sense to me since this is a dev artifact, i.e., if this stops working for some reason (particularly RTD changing domain scheme for PR builds) docs hosted everywhere else (local builds, docs.disnake.dev, disnake.rtfd.io etc etc) will still work as expected (and fixing such thing won't be hard either since this simply checks the domain name). If this is *really* preferred, i can do another check, but rn i see no reason in changing this.

Reply:
Done in [2aacbeb9708b3843d27ab8bf86e7ff5b5c39d9ef](https://github.com/DisnakeDev/disnake/pull/722/commits/2aacbeb9708b3843d27ab8bf86e7ff5b5c39d9ef)